kernel/src/fs/utils/inode.rs

```diff
@@ -80,6 +80,43 @@ impl From<DeviceType> for InodeType {
     }
 }
 
+bitflags! {
+    pub struct RequestPermission: u16 {
```

Comment:
The name `RequestPermission` may be too long since it appears frequently as a parameter in the trait function `check_permission()`. Additionally, treating these flags as "requests" might not be appropriate. Therefore, we can choose a more concise name such as `Permission` directly.